routes/api/users.js

```diff
@@ -4,7 +4,22 @@ const Joi = require("joi");
 const bcryptjs = require("bcryptjs");
 const jwt = require("jsonwebtoken");
 const User = require("../../models/userModel");
-const saltRounds = 10;
+const authMiddleware = require("../../middleware/authMiddleware");
+const multer = require("multer");
+const path = require("path");
+const jimp = require("jimp");
+const { v4: uuidv4 } = require("uuid");
```

Comment:
Warto pomyśleć nad tym żeby importy sortować sobie alfabetycznie i grupować to też dobrze wygląda i łatwiej się potem w tym odnaleźć. Można na przykład robić tak że na początku dajemy importy główne następnie importy zewnętrzne następnie importy własne. I dzięki temu mamy jasno i przejrzystą strukturę importów.